README.md

```diff
@@ -25,19 +25,20 @@ On release, automated continuous integration tests run the pipeline on a full-si
-10. Call SNPs and indels ([`GATK4 HaplotypeCaller`](https://gatk.broadinstitute.org/hc/en-us/articles/4409897180827-HaplotypeCaller))
-11. Merge multiple VCF files into one VCF ([`GATK4 MergeVCFs`](https://gatk.broadinstitute.org/hc/en-us/articles/4409924817691-MergeVcfs-Picard-))
-12. Index the VCF ([`GATK4 IndexFeatureFile`](https://gatk.broadinstitute.org/hc/en-us/articles/4409917499931-IndexFeatureFile))
-13. Filter variant calls based on certain criteria ([`GATK4 VariantFiltration`](https://gatk.broadinstitute.org/hc/en-us/articles/4409897204763-VariantFiltration))
-14. Present QC for raw read, alignment, gene biotype, sample similarity, and strand-specificity checks ([`MultiQC`](http://multiqc.info/), [`R`](https://www.r-project.org/))
+2. Read QC ([`FastQC v0.11.9`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
+3. Align reads to reference genome ([`STAR v2.7.9a`](https://github.com/alexdobin/STAR))
+4. Sort and index alignments ([`SAMtools v1.15`](https://sourceforge.net/projects/samtools/files/samtools/))
+5. Duplicate read marking ([`Picard MarkDuplicates v2.26.10`](https://broadinstitute.github.io/picard/))
+6. Splits reads that contain Ns in their cigar string ([`GATK v4.2.5.0 SplitNCigarReads`](https://gatk.broadinstitute.org/hc/en-us/articles/4409917482651-SplitNCigarReads))
+7. Estimate and correct systematic bias using base quality score recalibration ([`GATK v4.2.5.0 BaseRecalibrator`](https://gatk.broadinstitute.org/hc/en-us/articles/4409897206043-BaseRecalibrator), [`GATK v4.2.5.0 ApplyBQSR`](https://gatk.broadinstitute.org/hc/en-us/articles/4409897168667-ApplyBQSR))
+8. Convert a BED file to a Picard Interval List ([`GATK v4.2.5.0 BedToIntervalList`](https://gatk.broadinstitute.org/hc/en-us/articles/4409924780827-BedToIntervalList-Picard-))
+9. Scatter one interval-list into many interval-files ([`GATK v4.2.5.0 IntervalListTools`](https://gatk.broadinstitute.org/hc/en-us/articles/4409917392155-IntervalListTools-Picard-))
+10. Call SNPs and indels ([`GATK v4.2.5.0 HaplotypeCaller`](https://gatk.broadinstitute.org/hc/en-us/articles/4409897180827-HaplotypeCaller))
+11. Merge multiple VCF files into one VCF ([`GATK v4.2.5.0 MergeVCFs`](https://gatk.broadinstitute.org/hc/en-us/articles/4409924817691-MergeVcfs-Picard-))
+12. Index the VCF ([`GATK v4.2.5.0 IndexFeatureFile`](https://gatk.broadinstitute.org/hc/en-us/articles/4409917499931-IndexFeatureFile))
+13. Filter variant calls based on certain criteria ([`GATK v4.2.5.0 VariantFiltration`](https://gatk.broadinstitute.org/hc/en-us/articles/4409897204763-VariantFiltration))
+14. Annotate variants ([`snpEff v5.0`](https://pcingola.github.io/SnpEff/se_introduction/), [Ensembl VEP v104.3](https://www.ensembl.org/info/docs/tools/vep/index.html))
+15. Present QC for raw read, alignment, gene biotype, sample similarity, and strand-specificity checks ([`MultiQC v1.12`](http://multiqc.info/), [`R`](https://www.r-project.org/))
```

Comment:
I don't think having the tools version here is good for easy visibility.
Maybe adding it in a separate table?

Reply:
I thought about it earlier. Yes, a table is good. Do you prefer to have it as a markdown table or an image?

Reply:
md, that would be much easier to keep up to date